roles/monitoring_bbb/vars/main.yml

```diff
@@ -100,6 +110,19 @@ bbb_prometheus_scalelite_node_exporter_job:
   scrape_interval: "{{ bbb_monitoring_scrape_interval }}"
   scrape_timeout: "{{ bbb_monitoring_scrape_timeout }}"
 
+bbb_prometheus_owasp_node_exporter_job:
+  job_name: "{{ bbb_instance_name }}-owasp"
+  scheme: https
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+        - "{{ scalelite_host }}:9101"
+  basic_auth: "{{ bbb_prometheus_basic_auth }}"
+  relabel_configs: "{{ bbb_prometheus_owasp_relabel_configs }}"
```

Comment:
This could cause issues with duplicate metrics on the management and ALG servers, we should relabel the metrics in the scalelite jobs to have a different hostname

Reply:
Yes, having the owasp hostname as instance value for the management metrics would be confusing (I think they would be technically distinct though because of the job_name). Would a change to `replacement: "management.${1}"` in the relabel configs be sufficient or do you have a more elegant idea?

Reply:
Changed in relabel_config.